README.md

```diff
@@ -82,14 +82,15 @@ absl::variant<ConstraintInfo, std::vector<absl::Status>> P4ToConstraintInfo(
 ```C++
 /* p4_constraints/backend/interpreter.h */
 
-// Checks if a given table entry satisfies the entry constraint attached to its
-// associated table. Returns the empty string if this is the case, or a
-// human-readable nonempty string explaining why it is not the case otherwise.
-// Returns an InvalidArgument Status if the entry belongs to a table
-// not present in ConstraintInfo, or if it is inconsistent with the table
-// definition in ConstraintInfo.
+// Checks if a given table entry satisfies the entry/action(s) constraint(s)
+// attached to its associated table/action(s). Returns the empty string if this
+// is the case or a human-readable nonempty string explaining why it is not the
+// case otherwise. If the entry meets the table constraint and no action is
+// present, an empty string is returned. Returns an InvalidArgument Status if
```

Comment:
"If the entry meets the table constraint and no action is present, an empty string is returned."

We already said this one sentence earlier, remove?

Reply:
I see how you can infer this from the first sentence. Removed.